table/server/common/src/main/java/alluxio/table/common/udb/UdbFilterSpec.java

```diff
@@ -0,0 +1,505 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.table.common.udb;
+
+import alluxio.collections.Pair;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+/**
+ * Tables and partitions filtering specification.
+ */
+public final class UdbFilterSpec {
+  private static final Logger LOG = LoggerFactory.getLogger(UdbFilterSpec.class);
+  private static final UdbFilterSpec EMPTY_INSTANCE =
+      new UdbFilterSpec(Collections.emptySet(),
+          Mode.NONE,
+          Collections.emptyMap(),
+          Collections.emptySet(),
+          Mode.NONE);
+
+  /**
+   * Bypass/ignore entries.
+   */
+  public enum Entry {
+    BYPASS, IGNORE, NONE;
+  }
+
+  /**
+   * Inclusion/exclusion modes of tables and partitions.
+   */
+  public enum Mode {
+    INCLUDE, EXCLUDE, NONE;
```

Comment:
- same here, javadoc these enum items
- Maybe called `EntryMode` ?

Reply:
Renamed in 0c85e845413da76eefa233ded6e98076946dd94f. Added doc in cef77153696b4028a525829d6d17d898fa972e6c.